Hello-microverse-project | Pull Request

NicolasAndrehh opened the pull request:
# Milestone - Project Completed
In this pull request I implement the following features:

- Added a .html file that includes a `header` with the text "Hello Microverse!"
- Added a .css file that change the color of the header to `dark cyan`

NicolasAndrehh committed:
Update the ReadMe file

Delete some text of the readme file, text that was instructions from the template

IngDavidTM commented on README.md:
```diff
@@ -80,8 +77,6 @@ In order to run this project you need:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Nicolas Olaya**
 
 - GitHub: [@NicolasAndrehh](https://github.com/NicolasAndrehh)
```
- [x] Kindly don't forget to implement a [MIT license](https://choosealicense.com/licenses/mit/) because software license tells others what they can and can't do with your source code, so it's essential to make an informed decision. You can also set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). :+1:

NicolasAndrehh replied:
Okay, I'll do it, thanks for your suggestions!

IngDavidTM reviewed (changes requested):
## Hi @NicolasAndrehh  🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [IMPORTANT NOTE] Next time :next_track_button: note that you are not allowed to merge your branched unless you get approved by a code reviewer, It is an important step you should always follow at microverse. :+1: 

- [x]  As a general requirement you should remove the template logo and add Future Features section on the `README` file. This would make your repo looks more professional and this is one of the first things that hiring managers or your potential colleagues will check when considering you for a position on their team. To learn more you could read [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md)
Optionally, you can replace it with your own logo if you have one :+1:. 

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NicolasAndrehh commented:
# Hi @IngDavidTM!
- I'll do the changes you suggest, but just to be clear, i have to remove the microverse logo and also add a future features section on the readme file, is that right?

- And if that's right, it's ok to put "Soon I'll make a logo to put it in the ReadMe file" in the future features?

NicolasAndrehh committed:
Update to ReadMe file according to requirements

# ReadMe changes
- Remove the microverse template logo
- Add Future Features section

NicolasAndrehh committed:
Add MIT LICENSE to the project

- MIT LICENSE added

IngDavidTM commented:
Hi @NicolasAndrehh
Yes you have to remove the microverse logo, but is optional to create a new personal logo. :+1:

ABDULALI3468 reviewed (changes requested):
# Changes Required 👍 

Hi @NicolasAndrehh , 

<img src="https://media0.giphy.com/media/z3piokwf0WPH81MOhu/giphy.gif"/>

Although you have done great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> 1- Please consider adding the link to your LICENSE file in the README file. It was requested by the previous code reviewer. And it's a good practice for making a professional repository. You can achieve this by using the following syntax:
`[Text in this case MIT](link to the file)` 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: ![GitHub followers](https://img.shields.io/github/followers/ABDULALI3468?label=ABDULALI&style=social)
> * This is My **Linkedin**: [![Linkedin Badge](https://img.shields.io/badge/-ABDUL%20ALI-blue?style=flat-square&logo=Linkedin&logoColor=white&link=https://www.linkedin.com/in/abdul-ali-5400bb216/)](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@mrabdul_ali_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/mrabdul_ali)](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NicolasAndrehh committed:
Update ReadMe file

Add license to the ReadMe file

Gambit142 reviewed (approved):
Hi @NicolasAndrehh 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Good commit messages ✔️ 
- Descriptive Pull request summary ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
